site/pages/join-us/index.js

```diff
@@ -103,46 +80,84 @@ const apply = {
 const vacancies = {
   type: 'Title2',
   props: {
-    children: 'Current Vacancies',
+    children: <span className={styles.vacancyHeader}>Current Vacancies</span>,
   },
 };
 
-export default function JoinUs({ jobs }) {
-  return (
-    <div className={styles.background}>
+class HeaderContainer extends React.Component {
```

Comment:
Ideally this component would be in a separate file

Reply:
moved!